src/main/java/com/foundrysoftware/minecraft/plugins/PluginTemplate.java

```diff
@@ -34,4 +38,17 @@ public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command
         }
         return false;
     }
+    :Override
+    public boolean onInteract(PlayerInteractEvent e){
+        if(e.getItem()!=null && e.getItem().getItemMeta().getDisplayName().contains("score")) {
```

Comment:
so a few minor things
1) use "event" not e for the variable name same for "a" use armourStand.
2) make "score" into a class level constant ..... have you come across those?
4) make   if(e.getItem()!=null && e.getItem().getItemMeta().getDisplayName().contains("score")) { into a method that returns a boolean. You'll have to come up with a good name that descibes what is does aka hasTheWordScoreOnIt
5) make lines 44-50 into another method with a good name

Reply:
all of these comments are about code readability